CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[UPDATE]** Added `adyenEncryptedName` props on `TextField`
```

Comment:
This is not some very generic reusable API.
Ideally, I would prefer to have some extraInputAttributes API that can be used for adyen but as well any other weird cases in the future.
I can live with that too, your call.

Reply:
Yes we discussed this with @florianeyzat yesterday. I was thinking of having a sort of `extraInputAttributes` typed `InputHTMLAttributes<HTMLInputElement>` that we can spread safely on the input, so we totally control the DOM ouput of the `input` from the outside.
```
 <input {...extraInputAttributes} />
```

Reply:
Just adding my 2 cents: I agree with @wakooka, but, as `TextField` is basically a wrapper around the native input, I think it should _inherit_ the native inputs props and spread them on the native input.

We wouldn't have to do these PR anymore and this is the reason I added the "Use rest/spread operator" in the propositions to ease the Kirk/Kairos workflow.

Reply:
Hey guys, yes we discussed it with @wakooka, I'm working on a suitable solution here, a PR will come really soon, and I'm closing this one in the meantime